PascalABC.NET.SDK.Demo/PascalABC.NET.SDK.Demo.pasproj

```diff
@@ -0,0 +1,12 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="Current">
```

Comment:
Why not simply Project?

Reply:
I think it has some problems in either IDE or `dotnet build`. In any case, this will upgrade to a normal SDK-based project in the future, most likely.

Reply:
That should be IDE then. Because this syntax is looks antique. But that’s mostly nitpick